### ISSUE TITLE
Investigate usage of "Apache Camel Extension Pack" usage

### ISSUE THREAD

**Issue by apupier** [camel-tooling/vscode-camel-extension-pack#9]
@lhein reported a potential issue using this naming: https://github.com/camel-tooling/vscode-camel-extension-pack/pull/6#pullrequestreview-199653108

**Comment by apupier** [comment]
Lars suggested to have more insight from @davsclaus , so Claus?

**Comment by apupier** [comment]
that's right that "Apache Camel Extension Pack" might confuse users that it is an Apache project. Maybe we can change to "Extension Pack for Apache Camel"?

**Comment by davsclaus** [comment]
Yeah we can flip the name
